### PR TITLE
PRT-272 Send uri request over http not rpc

### PR DESCRIPTION
**Pull request by Aleksao998**
* **Please check if the PR fulfills these requirements**
- [x] Commit message follows the Contribution Guidelines
- [x] Tests ran locally and added/modified if needed
- [ ] Docs have been added/updated, if applicable
- [ ] If applicable - JIRA ticket ID was added

* **What kind of change does this PR introduce?** (Bug fix, feature, unit tests, docs update, ...)

Tendermint rpc has 3 types of calls
1. Tendermint URI HTTP
2. Tendermint RPC HTTP
3. Tendermint RPC Websockets

We had an issue where for tendermint URI we were converting the call to the rpc but in fact it should be sent as we do rest

* **What is the current behavior?** (You can also link to an open issue here)

If you send Tendermint URI Call it will convert it to the rpc. WIth this we lose some functionalities like default values

* **What is the new behavior (if this is a feature change)?**

Tendermint URI will be send as plain HTTP and RPC will send as RPC calls

* **Please describe what manual tests you ran, if applicable**

1. Run these commands and make sure they work
```
curl -X POST -H "Content-Type: application/json"  http://127.0.0.1:3341/1 --data '{"jsonrpc":"2.0","id":535439332833,"method":"block","params":["5"]}'

curl -X POST -H "Content-Type: application/json"  http://127.0.0.1:3341/1 --data '{"jsonrpc":"2.0","id":535439332833,"method":"status","params":[]}'

curl  http://127.0.0.1:3341/1/block?height=5

curl  http://127.0.0.1:3341/1/unconfirmed_txs (this one previously failed because default params were not working)
```

* **Other information**:

### CODE REVIEW

**Review by ranlavanet — CHANGES_REQUESTED** [review]

File: proto/spec/service_api.proto [hunk 1]
```diff
@@ -24,8 +24,9 @@ message Parsing {
 message ApiInterface {
   string interface = 1;
   string type = 2;
-  uint64 extra_compute_units = 3; 
-  SpecCategory category = 4;
+  uint64 extra_compute_units = 3;
+  BlockParser overwrite_block_parsing = 4;
```
[the remark this answers is not in the document]
Comment (ranlavanet) [reply proto/spec/service_api.proto:28]: dont forget to regenrate the proto files after

[the remark this answers is not in the document]
Comment (Aleksao998) [reply proto/spec/service_api.proto]: Fixed 52684875607c3536b995ae8cabe7f992d8df89f1

File: relayer/parser/parser.go [hunk 2]
```diff
@@ -251,98 +252,181 @@ func ParseCanonical(rpcInput RPCInput, input []string, dataSource int) ([]interf
...
-				if splitted[0] != prop_name || len(splitted) < 2 {
-					continue
-				} else {
-					retArr := make([]interface{}, 0)
-					retArr = append(retArr, splitted[1])
-					return retArr, nil
-				}
-			}
+		// If value attribute with propName exists in array return it
+		value := parseArrayOfInterfaces(unmarshalledDataTyped, propName, innerSeparator)
```
Comment (omerlavanet) [inline relayer/parser/parser.go:372]: well done!

**Review by omerlavanet — APPROVED** [review]
recent main changes to tendermint would not be introduced on the RUI path, need to manually merge

**Pull request merged**